app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/datamodel/Picture.kt

```diff
@@ -0,0 +1,12 @@
+package com.github.HumanLearning2021.HumanLearningApp.datamodel
+
+import com.google.firebase.firestore.DocumentReference
+
+/** Plain object for the database entity that represents a picture that learners can classify.
+ */
+data class Picture(
+    /** Location of the image in Cloud Storage. */
+    val url: String? = null,
+    /** The category that the picture depicts. (mandatory) */
+    val category: DocumentReference? = null,
```

Comment:
why DocumentReference and not Category object ?

Reply:
Then, when we load one image to the database, we would have to load the category as well, and handle duplication and re-serialization. That's cumbersome and inefficient.

Reply:
Actually I'll walk that back: it's possible to re-serialize nested objects using their primary key (see Firebase docs and blah).

As for efficiency: at my level of abstraction it makes sense to focus on one document at a time, but at the `model.DatabaseInterface` (or whatever it will be called) there will be a specific strategy to efficiently manage `Category` objects so we should be able to accomplish this theret